aries-site/src/layouts/content/AccessibilitySection.js

```diff
@@ -71,9 +71,19 @@ export const AccessibilitySection = ({ title, version }) => {
           title: successCriterion.title,
         };
 
+        // If the rule status is failed && level = AAA,
+        // change status to AAA failed
+        if (rule.status === 'failed' && extractedData.level === 'AAA') {
+          console.log('here aaa failed');
+          extractedData.status = 'AAA failed';
+        } else {
+          console.log('else');
```

Comment:
Don't forget to remove the console log statements

Reply:
removed